<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>skincntrl</class>
 <widget class="QWidget" name="skincntrl">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Beagleplayer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/BPres.qrc">
    <normaloff>:/res/beagleplayer_icon.png</normaloff>:/res/beagleplayer_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:blue;
color:white;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="songTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>title</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
       <item>
        <widget class="QLabel" name="cntrl_time">
         <property name="text">
          <string>0:00:00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="trackSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="PREV">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../res/BPres.qrc">
           <normaloff>:/res/RRWD_icon.png</normaloff>:/res/RRWD_icon.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RWD">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../res/BPres.qrc">
           <normaloff>:/res/RWD_icon.png</normaloff>:/res/RWD_icon.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="STOP">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../res/BPres.qrc">
           <normaloff>:/res/STOP_icon.png</normaloff>:/res/STOP_icon.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PAUSE">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../res/BPres.qrc">
           <normaloff>:/res/PAUSE_icon.png</normaloff>:/res/PAUSE_icon.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PLAY">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../res/BPres.qrc">
           <normaloff>:/res/FWD_icon.png</normaloff>:/res/FWD_icon.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="NEXT">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../res/BPres.qrc">
           <normaloff>:/res/FFWD_icon.png</normaloff>:/res/FFWD_icon.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="detach">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>detach</string>
         </property>
         <property name="icon">
          <iconset resource="../../res/BPres.qrc">
           <normaloff>:/res/UP_icon.png</normaloff>:/res/UP_icon.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LOOP">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../res/BPres.qrc">
           <normaloff>:/res/RPT_icon.png</normaloff>:/res/RPT_icon.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="volLayout"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res/BPres.qrc"/>
 </resources>
 <connections/>
</ui>
